ci.sh

```diff
@@ -19,22 +19,10 @@ if cargo clippy --version; then
 	cargo clippy --all-targets -- -D clippy::all -D clippy::cargo
 fi
 
-RUST_BACKTRACE=1 cargo build
-
-RUST_BACKTRACE=1 cargo build --all-features
-
-RUST_BACKTRACE=1 cargo build --target arm-unknown-linux-gnueabi
-RUST_BACKTRACE=1 cargo build --target armv7-unknown-linux-gnueabi
-RUST_BACKTRACE=1 cargo build --target armv7-unknown-linux-gnueabihf
-RUST_BACKTRACE=1 cargo build --target aarch64-unknown-linux-gnu
-RUST_BACKTRACE=1 cargo build --target i686-unknown-linux-gnu
-RUST_BACKTRACE=1 cargo build --target loongarch64-unknown-linux-gnu
-RUST_BACKTRACE=1 cargo build --target powerpc64-unknown-linux-gnu
-RUST_BACKTRACE=1 cargo build --target powerpc64le-unknown-linux-gnu
-RUST_BACKTRACE=1 cargo build --target riscv64gc-unknown-linux-gnu
-RUST_BACKTRACE=1 cargo build --target x86_64-pc-windows-msvc
-RUST_BACKTRACE=1 cargo build --target x86_64-apple-darwin
-RUST_BACKTRACE=1 cargo build --target aarch64-apple-darwin
-RUST_BACKTRACE=1 cargo build --target x86_64-unknown-freebsd
-
-RUST_BACKTRACE=1 cargo test
+cargo build
+
+cargo build --all-features
+
+cargo build --target "$TARGET"
+
+cargo test
```

Comment:
I would probably change this to run only if the arch triplet matches the system we run on. Either simply by running this when TARGET is `x86_64-unknown-linux-gnu` . Or somehow dynamically if we will use different runners.

Otherwise we run the same tests 13 times in the same configuration here.